src/Sarif.FunctionalTests/SarifConverterTestData/Fortify/bannedAPIs_m1.xml.sarif

```diff
@@ -1,19 +1,16 @@
 {
-  "version": "1.0.0-beta.2",
```

Comment:
Only that if you have an old file lying around you can tell what version wrote it. But I'm happy not to keep doing this.